Source/Particles/Resampling/VelocityCoincidenceThinning.cpp

```diff
@@ -24,6 +24,13 @@ VelocityCoincidenceThinning::VelocityCoincidenceThinning (const std::string& spe
         "Resampling min_ppc should be greater than or equal to 1"
     );
 
+    amrex::ParticleReal target_weight = 0;
+    if (utils::parser::queryWithParser(
+        pp_species_name, "resampling_algorithm_target_weight", target_weight
+    )) {
+        m_cluster_weight = target_weight * 2.0_prt;
```

Comment:
It might be worth adding a comment here that the factor of 2 is needed because each cluster is reduced to 2 particles.